functions.sh

```diff
@@ -67,14 +67,20 @@ function unsetTee {
 
 # Creates a unique results dir based on date, then links the common
 # results dir name to it.
+
 function setupResultsDir {
-    mkdir -p ${RESULTS_ARCHIVE_DIR}/${DATE}
+    mkdir -p ${RESULTS_ARCHIVE_DIR}/${DATE}/${RESULTS_SUB_DIR}
```

Comment:
RESULTS_SUB_DIR is define in cronjob.sh. My thought process was that once cronjob.sh determine if it is a benchmark or a test or even both, it will create the appropriate `dir`. This is why in cronjob.sh the function `setupResultsDir` is called in both cases (testing and benchmark)

Reply:
I see. In that case, there should probably be a default somewhere here, since `cronjob.sh` is a project-level script and these scripts are meant to be reused by different projects.

Reply:
The latest commits are not using this variable.